src/main/java/core/mate/academy/model/Truck.java

```diff
@@ -5,9 +5,33 @@
  * Do not remove no-field constructor
  */
 public class Truck extends Machine {
```

Comment:
According to the checklist, custom fields in classes should not be exposed directly and should be accessed through getters and setters. However, the `Truck` class does not provide any access modifiers for its fields `model` and `enginePower`. Consider adding private access modifiers to these fields to encapsulate the class's data.